src/Application.js

```diff
@@ -52,7 +82,12 @@ export class Application {
         soundVolume: false,
         musicVolume: false,
         voVolume: false,
-        sfxVolume: false
+        sfxVolume: false,
+        pointerSize: false,
+        controlSensitivity: false,
+        buttonSize: false,
+        removableLayers: false,
+        hudPosition: false
```

Comment:
For HUD position, I imagine the developer should be able to broadcast all of their supported docking locations. For instance, some games may only support top and bottom, and others all four docking locations. Currently `features` has always been declared as a boolean ("Yes, I support this feature"), but does it make sense to use an object here?

Reply:
We could keep the feature as a boolean so "Yes I support different layouts" and then do a second request to get them?

Reply:
For future reference, that's exactly what we decided:

```
app.container.respond('positions', ['top', 'bottom']);
```